open-craft/courses
# Address some feedback comments: notifications, test. And rename section

This includes some comments from OC-3352.

# Testing
1. check the diff. HTML shouldn't be broken
2. Create .tar.gz and import it into Studio, into a course with the following data: course:„onboarding“, org:„OpenCraft“, run:„course“
3. follow the course. Check that there's a new section about notifications, and that the section „configuring your opencraft e-mail“ was renamed to a longer name
4. check that the internal link added in „first technical task“ to „code quality“ works
5. check the content, it must address:
  - types of tests, how to run each, some are slow
  - handle notification overflow; synchronous/asynchronous and times
  - over-commiting; how to take tasks progressively during the week
  - communication about non-task related work
  - and in addition, the issues requested as part of the review in github
